model/roles.go

```diff
@@ -1632,40 +1632,40 @@ func validateColocatedContainerPortCollisions(RoleManifest *RoleManifest) valida
 
 	for _, role := range RoleManifest.Roles {
 		if len(role.ColocatedContainers) > 0 {
-			portsMap := map[int][]string{}
+			lookupMap := map[string][]string{}
 
 			// Iterate over this role and all colocated container roles and store the
-			// names of the roles for each port (there should be no list with more
-			// than one entry)
+			// names of the roles for each protocol/port (there should be no list with
+			// more than one entry)
```

Comment:
This works because we validate the protocol name, case-sensitively, before this happens, right?

Reply:
Yes, exactly. Now that you mention it, I forgot to point that out. I checked that only `TCP` or `UDP` (uppercase) are possible protocol content and therefore assume it will be fine to concatenate that with a slash to the port number.